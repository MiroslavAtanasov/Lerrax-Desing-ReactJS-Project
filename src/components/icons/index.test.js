import React from 'react'
import renderer from 'react-test-renderer'
import Icons from '.'
import TestingEnvironment from '../../test-utils/router'


describe('Icons component', () => {
    it('should render Icons component', () => {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    loggedIn: true,
                    id: '123'
                }
            }}>
                <Icons />
            </TestingEnvironment>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})