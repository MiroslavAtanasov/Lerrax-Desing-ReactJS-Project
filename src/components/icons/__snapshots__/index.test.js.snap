// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons component should render Icons component 1`] = `
<a
  className="social"
/>
`;
